### PR TITLE
Remove reek

### PR DESCRIPTION
**Pull request by zachmargolis**
See https://github.com/18f/identity-idp/pull/3547

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -59,16 +59,6 @@ plugins:
       - XXX
   pep8:
     enabled: true
-  reek:
-    enabled: true
-    exclude_patterns:
-    - 'spec/**/*'
-    - 'db/**/*'
-  rubocop:
```
Comment (jennywagg) [inline .codeclimate.yml:67]: In reference to my other comment - to be clear the idea is we're just getting rid of it in codeclimate, right? Keep it around in the app though?

Comment (zachmargolis) [reply .codeclimate.yml:67]: whoooops no I just meant to remove reek and keep rubocop 👮

Comment (jennywagg) [reply .codeclimate.yml:67]: Oh hehe

Comment (zachmargolis) [reply .codeclimate.yml:67]: Thanks for catching!

**Pull request merged**